src/app.rs

```diff
@@ -145,6 +132,11 @@ impl Application {
                     return Err(code);
                 };
 
+                info!(message = "Log level is enabled.", level = ?level);
```

Comment:
Did you mean to re-add these lines? They seem unrelated to the commit being reverted.

Reply:
This was moved from above, where it needed to happen in a `rt.block_on` when we had the span shipper as part of `trace::init`. Definitely looks a bit different though 😄

Reply:
Aha, yes I see where it was before 👍